crates/forge/tests/e2e/running.rs

```diff
@@ -717,7 +730,7 @@ fn validate_init(temp: &TempDir) {
             sierra = true
 
             [scripts]
-            test = "snforge test"
+            test = "snforge test"{appended_content}
```

Comment:
note: When initializing project with `snforge init`, we need to append `Scarb.toml` template to the expected value, in case of `scarb new` we keep `Scarb.toml` as it is.